core/main/src/processor/settings_processor.rs

```diff
@@ -232,7 +233,10 @@ impl SettingsProcessor {
                         true,
                         &ctx,
                         EVENT_ALLOW_PERSONALIZATION_CHANGED,
-                        None,
+                        Some(ContentListenRequest {
```

Comment:
Any way to clean this up a little? from a quick read, looks like the last key to set result "wins".  For example, if 3 keys are sent in the request to be set, the first one fails, and 2 and 3 succeed, is true to the correct return value?

Reply:
Current logic is the default for resp = true if there are issues in one of the entries it becomes false but the iteration goes on.